pkg/backup/backup.go

```diff
@@ -133,13 +133,11 @@ func getNamespaceIncludesExcludes(backup *api.Backup) *collections.IncludesExclu
 }
 
 type logger struct {
```

Comment:
Does this provide much value? Why not just use the logrus logger directly?

Reply:
@davecheney no, was just taking the smallest possible step towards logrus here. Plan is to get rid of this intermediate layer.